client_local_api/endpoints/health_check_test.go

```diff
@@ -0,0 +1,27 @@
+package endpoints
+
+import (
+	"github.com/stretchr/testify/assert"
+	"net/http/httptest"
+	"testing"
+	"time"
+)
+
+func TestHealthCheckReturnsExpectedJsonObject(t *testing.T) {
+
+	startupTime = time.Now()
+	currentTime = func() time.Time { return startupTime.Add(time.Minute) }
+
+	req := httptest.NewRequest("GET", "/irrelevant", nil)
+	resp := httptest.NewRecorder()
+
+	HealthCheckEndpoint(resp, req)
+
+	assert.JSONEq(t, `
```

Comment:
Lets have similar JSON formatting convention.

See `openvpn/service_discovery/dto/service_definition_test.go` Currently it was:
```
assert.JSONEq(
    t,
    `{
        "uptime" : "1m0s"
     }`,
    resp.Body.String()
```